runner/runner.go

```diff
@@ -2293,38 +2293,54 @@ func (r *Runner) HandleFaviconHash(hp *httpx.HTTPX, req *retryablehttp.Request,
 		if errCount == 2 {
 			break
 		}
-		URL, err := r.parseURL(potentialURL)
-		if err != nil {
-			continue
+		URL, err := urlutil.ParseURL(potentialURL, r.options.Unsafe)
+
+		isFavUrl, isBas64FavIcon := err == nil, false
```

Comment:
```suggestion
		isFavURL, isBase64FavIcon := err == nil, false
```

To maintain consistency and fix a typo.